qualifier/tests.py

```diff
@@ -123,6 +123,9 @@ async def test_multiple_staff_registration(self) -> None:
                 request.send.reset_mock()
 
                 await self.manager(create_request({"type": "staff.offduty", "id": id_}, request.receive, request.send))
+                
+                request.receive.assert_not_called()
+                request.send.assert_not_called()
```

Comment:
This should most likely also be copied to `test_staff_registration()`.

Reply:
Also added to `staff.offduty` step in `test_staff_registration` in 568c6b9